source/faq.txt

```diff
@@ -131,7 +131,8 @@ How do I view and modify my Privacy Settings?
 ---------------------------------------------
 
 To view and modify your |compass| privacy settings, from the top-level
-menu click :guilabel:`Help`, then :guilabel:`Privacy Settings`.
+menu click :guilabel:`Help` then click :guilabel:`Settings`, on the 
```

Comment:
[Suggestion]

Would it be helpful to make these steps into a numerical list?

Reply:
Thanks, I have updated to reflect this.